local-kubernetes/launch-environment.sh

```diff
@@ -2,9 +2,12 @@
 
 
 # Usage: Comment out the components you are working on
-kubectl apply -f local-sql-kubernetes-manifest.yml
+kubectl apply -f local-bapi-sql-kubernetes-manifest.yml
 kubectl apply -f local-bapi-kubernetes-manifest.yml
 kubectl apply -f local-azurite-kubernetes-manifest.yml
 kubectl apply -f local-dapi-kubernetes-manifest.yml
+kubectl apply -f local-isapi-sql-kubernetes-manifest.yml
+kubectl apply -f local-isapi-kubernetes-manifest.yml
+#kubectl apply -f local-isapi-sample-kubernetes-manifest.yml
```

Comment:
why commented out?

Reply:
Runs on same port as the actual isapi. I thought we want to keep this sample isapi service as a reference to how things can be wired up and not an actual service to stand up with everything else.